lib/runner/index.js

```diff
@@ -39,19 +40,31 @@ _.assign(Runner.prototype, {
      * @param {Function} callback
      */
     run: function (collection, options, callback) {
-        var runOptions = _.merge(_.omit(options, ['environment', 'globals', 'data']), this.options.run);
+        var self = this,
+            runOptions = _.merge(_.omit(options, ['environment', 'globals', 'data']), this.options.run);
 
         callback = backpack.normalise(callback);
         !_.isObject(options) && (options = {});
 
-        return callback(null, (new Run({
-            items: Runner.extractRunnableItems(collection, options.entrypoint),
-            data: Runner.normaliseIterationData(options.data, options.iterationCount),
-            environment: options.environment,
-            globals: _.has(options, 'globals') ? options.globals : this.options.globals,
-            certificates: options.certificates,
-            proxies: options.proxies
-        }, runOptions)));
+        Runner.extractRunnableItems(collection, options.entrypoint, function (err, runnableItems) {
```

Comment:
make callback of extract runnables pass two params.

Reply:
Addressed